redis/src/main/scala/zio/redis/codec/StringUtf8Codec.scala

```diff
@@ -0,0 +1,47 @@
+package zio.redis.codec
+
+import java.nio.charset.StandardCharsets
+
+import zio.redis.RedisError.CodecError
+import zio.schema.Schema
+import zio.schema.StandardType.StringType
+import zio.schema.codec.Codec
+import zio.stream.ZTransducer
+import zio.{ Chunk, ZIO }
+
+object StringUtf8Codec extends Codec {
```

Comment:
That codec is very poor, i've been considering to do primitive (short, long, etc.) codec 🤔  But we had agreement to do codec only for strings and i decided to stick with that

Reply:
It's fine for now, we can improve once the API is stabilized (or hopefully delete, once zio-schema supports it).